scripts/lintTs.mjs

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import * as os from 'os';
+import * as path from 'path';
+import * as url from 'url';
+import chalk from 'chalk';
+import {ESLint} from 'eslint';
+import pLimit from 'p-limit';
+import {getPackagesWithTsConfig} from './buildUtils.mjs';
+
+// we want to limit the number of processes we spawn
+const cpus = Math.max(1, os.cpus().length - 1);
+const mutex = pLimit(cpus);
+
+const fix = process.argv.slice(2).some(arg => arg === '--fix');
+
+const monorepoRoot = path.resolve(url.fileURLToPath(import.meta.url), '../..');
+
+const packagesWithTs = getPackagesWithTsConfig()
+  .map(({packageDir}) => packageDir)
+  .concat(path.resolve(monorepoRoot, 'e2e'))
+  // TODO: run all all projects
```

Comment:
it's probably better with an allow list and deny list - that way we can fix one specific package instead of going one by one